modules/installation-requirements-user-infra.adoc

```diff
@@ -173,7 +173,7 @@ ifdef::ibm-z[]
 ^[1]^ 1 physical core (IFL) provides 2 logical cores (threads) when SMT-2 is enabled. The hypervisor can provide 2 or more vCPUs.
 endif::ibm-z[]
 ifndef::ibm-z[]
-^[1]^ 1 physical core provides 2 vCPUs when hyper-threading is enabled. 1 physical core provides 1 vCPU when hyper-threading is not enabled.
+^[1]^ 1 vCPU is equivalent to 1 physical core when HyperThreading/SMT is not enabled. When enabled, you should use the formula '(Threads x Cores) x Sockets = vCPUs' to calculate the corresponding ratio.
```

Comment:
This would be the first appearance of "SMT" in the module (and the assembly, for that matter), so suggest spelling it out and matching the case elsewhere in the same assembly:

> 1 vCPU is equivalent to 1 physical core when simultaneous multithreading (SMT), or hyperthreading, is not enabled.

Reply:
Suggestion (uses multiplication sign `×` instead of `x` per IBMSG, drops "you should", and lowercases items in the formula):

> When enabled, use the following formula to calculate the corresponding ratio: (threads × cores) × sockets = vCPUs.

Reply:
Consider updating this formatting per [Alternative footnote styling in tables](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#alternative-footnote-styling-in-tables) to better mimic Asciidoctor's styling (since these are kinda fake DIY footnotes).